perfkitbenchmarker/providers/gcp/gce_network.py

```diff
@@ -35,11 +35,12 @@
 class GceFirewallRule(resource.BaseResource):
   """An object representing a GCE Firewall Rule."""
 
-  def __init__(self, name, project, port):
+  def __init__(self, name, project, port, network):
     super(GceFirewallRule, self).__init__()
     self.name = name
     self.project = project
     self.port = port
+    self.network = network
 
   def __eq__(self, other):
```

Comment:
It doesn't need it right now, because we already compare projects, and  we only have one Network per project, but that  could change. I'll add it.